.github/FUNDING.yml

```diff
@@ -0,0 +1,3 @@
+github: linuxserver
+open_collective: linuxserver
+custom: ["https://patreon.com/overseerr",]
```

Comment:
Maybe use the github sponsor link, as it has less fees?

Reply:
I'll add the gh sponsor link as well so it'll show both.